lib/smart_answer_flows/maternity-paternity-pay-leave.rb

```diff
@@ -1,9 +1,9 @@
 module SmartAnswer
-  class PayLeaveForParentsFlow < Flow
+  class MaternityPaternityPayLeaveFlow < Flow
     def define
       start_page_content_id "1f6b4ecc-ce2c-488a-b9c7-b78b3bba5598"
       flow_content_id "177cde4d-e52f-4629-bbbe-ec85a18ed944"
-      name "pay-leave-for-parents"
+      name "maternity-paternity-pay-leave"
```

Comment:
I think that since the content ID is the same, the Publishing API will automatically set up the redirect for this, which is nice.

Reply:
This was my hope 🤞